.github/workflows/cypress.yaml

```diff
@@ -0,0 +1,52 @@
+name: Cypress Tests
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v3
+        with:
+          python-version: '3.8'
+          cache: 'pip'
+      - name: install dependencies
+        run: |
+          pip install poetry
+          poetry install
+      - name: set config
+        env:
+          SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
+          SNOWFLAKE_USERNAME: ${{ secrets.SNOWFLAKE_USERNAME }}
+          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
+        run: |
+          mkdir $HOME/.snowsql
+          envsubst < deploy/opscenter.config > $HOME/.snowsql/config
+      - name: set env
+        run: |
+          echo "OPSCENTER_PACKAGE=PACKAGE_DEV" >> "$GITHUB_ENV"
+          echo "OPSCENTER_DATABASE=\"DATABASE_OPSCENTER_DEV\"" >> "$GITHUB_ENV"
+          echo "OPSCENTER_APP=APP_OPSCENTER_DEV" >> "$GITHUB_ENV"
+      - name: run deploy
+        env:
+          OPSCENTER_PACKAGE: ${{env.OPSCENTER_PACKAGE}}
+          OPSCENTER_DATABASE: ${{env.OPSCENTER_DATABASE}}
+          OPSCENTER_APP: ${{env.OPSCENTER_APP}}
+        run: "poetry run python deploy/deploy.py"
```

Comment:
doesn't this have to do a devdeploy?

Reply:
I based it on main.yaml, assumed it did a dev server.  I guess I need to create a .snowsql config and run the steps for local dev?

Reply:
you should be able to re-use the existing config...I think...and just switch it to do a dev deploy and run streamlit